app/controllers/people_controller.rb

```diff
@@ -40,10 +40,13 @@ def person
     @person ||= Person.find(params[:person_id])
   end
 
-  def export
-    odt_file = Odt::Cv.new(entry).export
+  # rubocop:disable Metrics/LineLength
+  def export(anon = params[:anon].presence || 'false')
+    odt_file = Odt::Cv.new(entry).export(anon != 'false')
     send_data odt_file.generate,
               type: 'application/vnd.oasis.opendocument.text',
-              disposition: content_disposition('attachment', filename(entry.name, 'cv'))
+              disposition: content_disposition('attachment',
+                                               anon != 'false' ? 'cv.odt' : filename(entry.name, 'cv'))
```

Comment:
The name of the file should be anonymized_cv.odt according to the DoD.
Also refactor this filename to a variable. You need to make ruby code readable by naming things and creating sentences using this naming. If you need to declare a variable to do so, do it. Juggling around with ternary and other things only makes it more complicated and less readable.